05.NablaTutorialExample/main.cpp

```diff
@@ -167,7 +167,7 @@ class NablaTutorialExampleApp : public ApplicationBase
 		const auto swapchainImageUsage = static_cast<asset::IImage::E_USAGE_FLAGS>(asset::IImage::EUF_COLOR_ATTACHMENT_BIT);
 		const video::ISurface::SFormat surfaceFormat(asset::EF_R8G8B8A8_SRGB, asset::ECP_COUNT, asset::EOTF_UNKNOWN);
 
-        CommonAPI::InitWithDefaultExt(initOutput, video::EAT_VULKAN, "NablaTutorialExample", WIN_W, WIN_H, SC_IMG_COUNT, swapchainImageUsage, surfaceFormat, nbl::asset::EF_D32_SFLOAT);
+        CommonAPI::InitWithDefaultExt(initOutput, video::EAT_VULKAN, "NablaTutorialExample", FRAMES_IN_FLIGHT, WIN_W, WIN_H, SC_IMG_COUNT, swapchainImageUsage, surfaceFormat, nbl::asset::EF_D32_SFLOAT);
```

Comment:
Please change all example's API types to `EAT_VULKAN` and test:
- No crashes should happen
- No validation errors (will print as red text) regarding new changes for buffer api
Keep in mind some examples didn't use to work on vulkan so keep them as is but test on vulkan (Please refer to the table I provided for you previously on examples statuses)